lib/dry/container/mixin.rb

```diff
@@ -183,6 +183,22 @@ def each_key(&block)
         self
       end
 
+      # Calls block once for each key/value pair in the container, passing the key and the registered item parameters.
+      #
+      # If no block is given, an enumerator is returned instead.
```

Comment:
This statement is not true with the current code as `#each` returns `self`, it should probably just return the result of `Resolver#each`